clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/KafkaInputFormat.java

```diff
@@ -56,8 +55,10 @@ protected Map<PubSubTopicPartition, Long> getLatestOffsets(JobConf config) {
     TopicManagerContext topicManagerContext =
         new TopicManagerContext.Builder().setPubSubPropertiesSupplier(k -> consumerProperties)
             .setPubSubTopicRepository(pubSubTopicRepository)
-            .setPubSubAdminAdapterFactory(new ApacheKafkaAdminAdapterFactory())
-            .setPubSubConsumerAdapterFactory(new ApacheKafkaConsumerAdapterFactory())
+            .setPubSubAdminAdapterFactory(
+                PubSubClientsFactory.fromVeniceProperties(consumerProperties).getAdminAdapterFactory())
```

Comment:
Let's create one `PubSubClientsFactory` and then get both admin and consumer factory out of it.